wercker.yml

```diff
@@ -14,8 +14,8 @@ build:
         - script:
             name: install glide
             code: |
-              curl --location -O  https://github.com/Masterminds/glide/releases/download/v0.11.1/glide-v0.11.1-linux-amd64.tar.gz
-              tar zxf glide-v0.11.1-linux-amd64.tar.gz
+              curl --location -O  https://github.com/Masterminds/glide/releases/download/v0.12.3/glide-v0.12.3-linux-amd64.tar.gz
```

Comment:
I forgot to mention this, but I added an environment variable to wercker for thingfulx called: $GLIDE_VERSION, such that rather than having to change this all the time, we can just change that variable to install different versions (I know I know, `dep` is the future ;).

This would mean changing both `v0.12.3` instances here to `${GLIDE_VERSION}`.